shinken/commandcall.py

```diff
@@ -132,17 +124,6 @@ def __getstate__(self):
             if hasattr(self, prop):
                 res[prop] = getattr(self, prop)
 
-        # The command is a bit special, we just put it's name
```

Comment:
I guess here you remove the possibility the command is a string so that you don't need late linkify?